src/Terminal/commands/ls.tsx

```diff
@@ -139,10 +139,12 @@ export function ls(
       }),
     )();
 
-    const rowSplit = row
-      .split(" ")
-      .map((x) => x.trim())
-      .filter((x) => !!x);
+    console.log(row);
```

Comment:
Are this `console.log` and the one below supposed to be merged or were they for debugging?

Reply:
That should be removed, will do.